options.go

```diff
@@ -134,6 +135,7 @@ type Options struct {
 	signatureData      *SignatureData
 	oidcVerifier       *oidc.IDTokenVerifier
 	jwtBearerVerifiers []*oidc.IDTokenVerifier
+	realClientIPParser realClientIPParsder
```

Comment:
```suggestion
	realClientIPParser realClientIPParser
```

Reply:
That's what I get for relying on code completion 😅